predictions_FB.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from fbprophet import Prophet
+import yfinance as yf
+import math
+import os
+import sys
+import pandas as pd
+pd.options.mode.chained_assignment = None
+
+
+class suppress_stdout_stderr(object):
+    '''
+    Used to supress the verbose output of the inner workings of the FB prophet model
+    '''
+    def __init__(self):
+        self.null_fds = [os.open(os.devnull, os.O_RDWR) for x in range(2)]
+        self.save_fds = (os.dup(1), os.dup(2))
+
+    def __enter__(self):
+        os.dup2(self.null_fds[0], 1)
+        os.dup2(self.null_fds[1], 2)
+
+    def __exit__(self, *_):
+        os.dup2(self.save_fds[0], 1)
+        os.dup2(self.save_fds[1], 2)
+        os.close(self.null_fds[0])
+        os.close(self.null_fds[1])
+
+def fb_predict(dfi, plot = False, days =1):
+    '''
+    This function takes in:
+    dfi - A dataframe with just the column of close price named as Close and date as index 
+    plot - Whether you want to plot the resulyts or not( Default is False)
+    days - Number of days into the future you want to predict (Dafault is 1)
+
+    This function returns:
+    mean_err - The mean percentage error in calculations
+    pred - A pandas datafrmae with predictions, upper and lower bounds
+
+    '''
```

Comment:
Good job documenting the method! If you use PyCharm and initialize a multiline comment (`doc string`), it'll create the fields for documentation for you!

The benefit of doing that is that you can access the documentation of a function using the `__doc__` property. So for `fb_predict`, you can access the documentation for this method by `fb_predict.__doc__`

Typically, you'd want to add `@:param` for each parameter and `@:return` to describe what is being returned.

[This article](https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html) should help you get started.

Reply:
Done, have a look and let me know if it looks good now.